test/system/203-network.bats

```diff
@@ -24,9 +24,8 @@ load 'libs/helpers'
 readonly RESOLVER_PYTHON3='\
 import socket; \
 import sys; \
-family = socket.AddressFamily.AF_INET if sys.argv[1] == "A" else 0; \
-family = socket.AddressFamily.AF_INET6 if sys.argv[1] == "AAAA" else 0; \
```

Comment:
Aha, `family` was getting overwritten to `0` in the second line if `sys.argv[1]` is `"A"`, right?